docs/api/bluetooth/GattServer.md

```diff
@@ -14,6 +16,16 @@ You must access values of the characteristic and the characteristic descriptor p
 
 You can query the server by invoking the function `areUpdatesEnabled()` to find out if a client has subscribed to a given characteristic's value update.
 
+#### Attribute Protocol Maximum Transmission Unit (ATT_MTU)
```

Comment:
Because this section is an exact duplicate, can we instead replace it with a link to the relevant section in GattClient.md?

Reply:
OK. The client version is the fuller one as this one shortens the paragraph about what the client does. How do I link?

Reply:
The link to GattClient is "../apis/gattclient.html"